client/src/components/Navbar/useStyles.ts

```diff
@@ -0,0 +1,72 @@
+import { makeStyles } from '@mui/styles';
+import { Theme } from '@mui/material/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    backgroundColor: 'white !important',
+  },
+  toolbar: {
+    boxShadow: '1px 1px 5px lightgrey, -1px -1px 5px lightgrey',
+    height: '10vh',
+  },
+  login: {
+    width: 120,
+    height: 45,
+    borderRadius: theme.shape.borderRadius,
+    fontSize: 16,
+    backgroundColor: 'white !important',
```

Comment:
You can pass a `color` prop to the MUI Button (also make sure we set the appropriate colors within MUI's theme)